src/main/cpp/tests/CMakeLists.txt

```diff
@@ -85,3 +85,6 @@ ConfigureTest(PARSE_URI
 ConfigureTest(SUBSTRING_INDEX
     substring_index.cpp)
 
+ConfigureTest(HLLPP
```

Comment:
Please make sure to run this to verify if there is any issue:
```
compute-sanitizer --tool memcheck gtests/HLLPP --rmm_mode=cuda
```

Reply:
Done:
```
========= COMPUTE-SANITIZER
Running main() from /home/chongg/code/spark-rapids-jni/target/jni/cmake-build/_deps/gtest-src/googletest/src/gtest_main.cc
[==========] Running 2 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 2 tests from HyperLogLogPlusPlusUDFTest
[ RUN      ] HyperLogLogPlusPlusUDFTest.Reduction
[       OK ] HyperLogLogPlusPlusUDFTest.Reduction (424 ms)
[ RUN      ] HyperLogLogPlusPlusUDFTest.Groupby
[       OK ] HyperLogLogPlusPlusUDFTest.Groupby (134 ms)
[----------] 2 tests from HyperLogLogPlusPlusUDFTest (558 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 1 test suite ran. (558 ms total)
[  PASSED  ] 2 tests.
========= ERROR SUMMARY: 0 errors

```